script/reviewer.go

```diff
@@ -19,11 +20,84 @@ func (r *Reviewer) Review() {
 
 	r.application.Parse(r.gitHub.Issue)
 
-	if isTestingIssue() {
-		if r.application.IsValid() {
-			debugMessage("Application has no problems")
-		} else {
-			debugMessage("Application problems:", r.application.RenderProblems())
+	status := r.getStatus()
+	r.updateLabels(status)
+	r.createComment(status)
+}
+
+func (r *Reviewer) getStatus() string {
+	if *r.gitHub.Issue.State == "closed" {
+		return "closed"
+	} else if r.gitHub.IssueHasLabel(LabelStatusApproved) {
+		return "approved"
+	} else if r.gitHub.IssueHasLabel(LabelStatusReviewing) {
+		return "reviewing"
+	} else if r.gitHub.IssueHasLabel(LabelStatusInvalid) {
+		return "invalid"
+	} else {
+		return "new"
+	}
+}
+
+func (r *Reviewer) createComment(status string) {
+	title := ""
+	body := ""
+	details := fmt.Sprintf("<details>\n<summary>Application data...</summary>\n\n```json\n%s\n```\n</details>", r.application.GetData())
+
+	if status == "closed" {
+		body = "Oops! This application is closed can no longer be processed. If this is an error, please reach out to [opensource@1password.com](mailto:opensource@1password.com)."
```

Comment:
```suggestion
		body = "Oops! This application is closed and can no longer be processed. If this is an error, please reach out to [opensource@1password.com](mailto:opensource@1password.com)."
```

1. I don't like the _Oops!_. It does not feel in line with our normal tone.
2. Our style guide would have you [omit the please](https://support.1password.com/style-guide/#please).
3. Rather than [reach out to](https://support.1password.com/style-guide/#colloquialisms-and-phrasal-verbs), say _contact us at_.

Similar applies to the _"approved"_ body.